src/data/hi.yml

```diff
@@ -479,32 +493,32 @@ community:
   sharing3: "के तहत लाइसेंस प्राप्त है। कृपया बेझिझक साझा करें और आरोपण के साथ रीमिक्स करें। "
 
   contribute-title: "योगदान"
-  contribute1: "हमारा समुदाय हमेशा सभी अलग-अलग तरीकों से मदद करने के लिए उत्साही लोगों की तलाश में रहता है।"
+  contribute1: "हमारा समुदाय हमेशा उत्साही लोगों की तलाश में रहता है जो हर तरह से मदद करें।"
   develop-title: "विकास करना।"
   develop1: "GitHub"
-  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों को प्रलेखित किया जाता है, और कोड के बारे में चर्चा होती है। आरंभ करने के लिए "
+  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों का दस्तावेजीकरण किया जाता है, और कोड के बारे में चर्चा की जाती है। आरंभ करने के लिए "
   develop3: " विकास शिक्षण"
   develop4: " देखें, या "
   develop5: "अपना स्वयं का लाइब्रेरी बनाएं।"
   document-title: "दस्तावेज़।"
-  document1: " दस्तावेज़ीकरण का शौक सभी को होता है। "
+  document1: " सभी को दस्तावेज़ीकरण पसंद है। मदद की जरूरत है "
```

Comment:
If this is related to docs or documentation please find a better word, I think there should be something better than "daataavez"

Reply:
प्रलेखन is  one alternate word for documentation other than "daataavez". So should I change it to प्रलेखन